packages/website/src/pages/_layout.tsx

```diff
@@ -9,16 +9,11 @@ type RootLayoutProps = { children: ReactNode };
 
 export default async function RootLayout({ children }: RootLayoutProps) {
   return (
-    <html>
-      <head></head>
-      <body>
-        <Providers>
-          <Meta />
-          <div id="__waku">{children}</div>
-          <Analytics />
-        </Providers>
-      </body>
-    </html>
+    <Providers>
+      <Meta />
+      <div id="__waku">{children}</div>
```

Comment:
this is just used for the website, right? 

It reminds me of `<div id="__next" />` which is the root element nextjs uses for their react app. 

Would we be open to maybe renaming this? sorry for the nit

Reply:
> It reminds me of `<div id="__next" />`

This is exactly the one for Waku, as I understand.

Reply:
is this the only place we reference this id though?

 https://github.com/dai-shi/waku/blob/main/packages/website/src/styles.css#L12-L19

Reply:
yeah, not forcing it at the moment. never mind. though, actually, i'm curious what was your suggestion about renaming.

Reply:
I guess my main question is just for why we're using "__waku" here. I shouldn't suggest changing it before understanding why it's used.

For now it seems like it is only used for styling in which case, I would think a class name would make more sense. But I am probably just missing something.

Reply:
I'm not sure either. Can also be a historical reason.

Reply:
not a big deal either way, thanks 👍